manifests/instance.pp

```diff
@@ -504,6 +510,9 @@
 
   $bind_arr = [$bind].flatten
 
+  $default_redis_version = '8.0.0'
```

Comment:
Do we really need the version magic? 
Version facts are quite opaque and lead to non-idempotent behaviour

Reply:
those new parameters are from version 7.0.0, so I was looking for a way to keep old setup working, and providing a new config file for setup with redis version > 7.0

Reply:
So these configs are available in redis 7.x? Anything below 7.4 is EOL so we could just drop the version condition completely

Reply:
I see repl-diskless-sync and latency-tracking in here https://raw.githubusercontent.com/redis/redis/7.4/redis.conf